seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java

```diff
@@ -854,4 +854,12 @@ default String getCollateSql(String collate) {
     default String dualTable() {
         return "";
     }
+
+    default Optional<String> getDefaultDatabase() {
+        return Optional.empty();
+    }
+
+    default boolean useThreePartTablePath() {
+        return false;
+    }
```

Comment:
Why not reuse https://github.com/apache/seatunnel/blob/cbb777e637d898722270b84fe87e522dc5f2c987/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java#L296?